lib/index.js

```diff
@@ -69,11 +74,12 @@ AdWords.prototype.page = function() {
  */
 
 AdWords.prototype.track = function(track) {
+  var remarketing = this.options.remarketing;
   var id = this.options.conversionId;
   var events = this.events(track.event());
   var revenue = track.revenue() || 0;
   each(function(label) {
-    var props = track.properties();
+    var props = remarketing ? extend(track.properties(), window.google_custom_params) : {};
```

Comment:
@hankim813 this is what I meant - we would send `track.properties` earlier but now we only send it if `remarketing` is enabled.